website/client/components/chat/chatCard.vue

```diff
@@ -1,8 +1,9 @@
 <template lang="pug">
 div
   .mentioned-icon(v-if='isUserMentioned')
-  .message-hidden(v-if='!inbox && msg.flagCount === 1 && user.contributor.admin') Message flagged once, not hidden
-  .message-hidden(v-if='!inbox && msg.flagCount > 1 && user.contributor.admin') Message hidden
+  .message-hidden(v-if='!inbox && user.contributor.admin && msg.flagCount === 1') Message flagged once, not hidden
```

Comment:
these three `message-hidden ` might become a bit more readable if we could use only one and then display the string based on a computer prop that returns the string maybe?

Reply:
I think I've done that how you wanted it but if it's not exactly what you had in mind, tell me and I'll change it. https://github.com/HabitRPG/habitica/pull/11239/commits/180ae50fb74444a70a8e0ff639890ee280c5e182
You were right - it's nicer now!

The new code has a check for msg.flagCount being non-zero in two places (in the `.message-hidden(v-if` line and in `flagCountDescription()`) which is redundant but I think it's clearer this way. However I'm happy to remove the check from either location if you want.

I still need to create the mandrill template. I'll post again when that's done. Feel free to ignore this until then if you want. :)